importer/context.go

```diff
@@ -139,6 +140,26 @@ func (ic *importContext) Run() error {
 	// nolint
 	sh.WriteString("#!/bin/sh\n\n")
 
+	dcfile, err := os.Create(fmt.Sprintf("%s/databricks.tf", ic.Directory))
```

Comment:
i think you've forgot to surround it with flag :)

Reply:
everybody needs it! :-)

Reply:
nope. that would break tf0.12

Reply:
I think that we can generate it based on the Terraform version - I need to check if it's possible to get it from the inside of the provider